src/App.js

```diff
@@ -1,12 +1,51 @@
+import { useDispatch, useSelector } from 'react-redux';
 import './App.css';
-import { Route, Routes } from 'react-router-dom';
+import { Navigate, Route, Routes } from 'react-router-dom';
+import { useEffect } from 'react';
+import Login from './pages/auth/Login/Login';
+import AuthLayout from './pages/auth/Layout/Layout';
+import ProtectedRoute from './components/ProtectedRoute/ProtectedRoute';
+import HomePage from './pages/Home/Home';
+import Register from './pages/auth/Register/Register';
+import ErrorMessage from './components/messages/ErrorMessage';
+import NoticeMessage from './components/messages/NoticeMessage';
+import { clearAll } from './slices/appSlice/appSlice';
 
 function App() {
+  const alert = useSelector((state) => state.app.alert);
+  const notice = useSelector((state) => state.app.notice);
+  const dispatch = useDispatch();
+  // remove the alert and notice after 10 seconds
+  useEffect(() => {
+    if (alert || notice) {
+      setTimeout(() => {
+        dispatch(clearAll());
+      }, 10000);
+    }
+  });
   return (
-    <div className="App">
+    <div className="App font-primary">
+      {alert && <ErrorMessage message={alert} />}
+      {notice && <NoticeMessage message={notice} /> }
       <Routes>
-        <Route path="/" element={<h1 className="text-3xl text-red-500 font-bold underline">Gvash planes</h1>} />
+        <Route path="/" element={<HomePage />} />
+        {/* guests routes */}
+        <Route element={<AuthLayout />}>
+          <Route path="/login" element={<Login />} />
+          <Route path="/register" element={<Register />} />
+        </Route>
+        {/* protected routes aka users routes */}
+        <Route element={<ProtectedRoute />}>
+          {/* this is the example route and usage you can find how to use it in that component */}
+          <Route path="/user" element={<HomePage />} />
```

Comment:
- [ ] Kindly make this home route public as suggested by @V-Blaze before because it should be accesseble to all users

Reply:
This one is an example route if you go to `/` you can see what I wrote in comments